pkgs/applications/misc/gnuradio-ais/default.nix

```diff
@@ -26,11 +26,15 @@ stdenv.mkDerivation rec {
     done
   '';
 
+  patches = [ ./fix-volk.patch ];
```

Comment:
They might accept straight forward build-fixes though. In the worst case, you still raise awareness and somebody can profit from the changes.